Sources/XiEditor/RPCSending.swift

```diff
@@ -19,6 +19,10 @@ import AppKit
 /// These logs can be used for profiling & debugging.
 let XI_RPC_LOG = "XI_CLIENT_RPC_LOG"
 
+let NEW_LINE = [0x0a as UInt8]
+let SEND_LOG_PREFIX = "[CLIENT] ".data(using: .utf8)!
+let RECV_LOG_PREFIX = "[CORE]   ".data(using: .utf8)!
```

Comment:
Perhaps change these to `CORE` and `CLIENT` too? Then I think this is good to go.